### DISCUSSION TITLE
Proposed contribution weights for 1.14.7 release

### DISCUSSION THREAD

**Discussion by patricklodder** [dogecoin/dogecoin#3505] [General]
Please find a long table below where I have assigned weights to the contributions for the 1.14.7 release, helping the different tipjars and funds out there to come to a payout. Fortunately for the software but unfortunately for the table below, there were many more co-authors and reviewers per pull request this release, so I couldn't do this in a wide format this time. I've left the table in the order I have scored them: first the large contributions, then normal sized contributions and then the trivial/small stuff.

Like last time, I'm weighting large contributions as 25, small/trivial as 1 and the rest as 5.

Please let me know if you have any questions about or corrections to this.

| PR | Contributor | Role | Weight | Description/Comment
| ---: | :--- | :--: | ---: | :---
| #3204 | xanimo | Co-Author | 25 | Major update to binary inspection tooling
| #3204 | patricklodder | Co-Author | 25 | 
| #3204 | chromatic | Reviewer | 25 | 
| #3222 | xanimo | Co-Author | 25 | Major update to build process
| #3222 | patricklodder | Co-Author | 25 | 
| #3222 | chromatic | Reviewer | 25 | 
| #3222 | michilumin | Reviewer | 25 | 
| #3222 | edtubbs | Reviewer | 25 | 
| #3415 | patricklodder | Author | 25 | Major security fixes to Qt
| #3415 | chromatic | Reviewer | 25 | 
| #3415 | slightlyskepticalpotat | Reviewer | 5 | Local build test only
| #2773 | patricklodder | Author | 5 | Experimental feature (sse2 scrypt)
| #2773 | edtubbs | Reviewer | 5 | 
| #2773 | victorsk2019 | Reviewer | 5 | 
| #2773 | chromatic | Reviewer | 5 | 
| #3060 | nhanaz | Author | 5 | Vietnamese translation
| #3060 | mintodev | Reviewer | 5 | Language review
| #3025 | edtubbs | Author | 5 | CI enhancement
| #3025 | patricklodder | Reviewer | 5 | 
| #3025 | chromatic | Reviewer | 5 | 
| #3081 | sipa | Author | 5 | Custom RIPEMD160 implementation in python
| #3081 | xanimo | Author | 5 | Backport author
| #3081 | patricklodder | Reviewer | 5 | 
| #3082 | patricklodder | Author | 5 | Subtree update (secp256k1)
| #3082 | chromatic | Reviewer | 5 | Gitian build
| #3082 | xanimo | Reviewer | 5 | Gitian build
| #3071 | edtubbs | Author | 5 | Experimental feature (avx2 for windows)
| #3071 | chromatic | Reviewer | 1 | Shasum check only
| #3071 | michilumin | Reviewer | 5 | 
| #3079 | practicalswift | Author | 5 | Bugfix for openssl 3 errors
| #3079 | patricklodder | Author | 1 | Backport author
| #3079 | chromatic | Reviewer | 5 | 
| #3079 | michilumin | Reviewer | 5 | 
| #3079 | edtubbs | Reviewer | 5 | 
| #3084 | chromatic | Author | 5 | RPC enhancement (redo of #2942)
| #3084 | patricklodder | Reviewer | 1 | 
| #3088 | junderw | Author | 5 | Bugfix on -maxtxfee
| #3088 | michilumin | Reviewer | 5 | 
| #3088 | chromatic | Reviewer | 5 | 
| #3091 | edtubbs | Author | 1 | Experimental feature, reverted because broke non-experimental. Reworked in #3188
| #3091 | michilumin | Reviewer | 1 | 
| #3103 | randomwalk266 | Author | 5 | zh_CN translation
| #3103 | bobdos | Reviewer | 5 | 
| #3103 | slightlyskepticalpotat | Reviewer | 5 | 
| #3107 | slightlyskepticalpotat | Author | 5 | Snap packaging
| #3107 | chromatic | Reviewer | 5 | 
| #3080 | xanimo | Author | 5 | Update python ltc-scrypt integration
| #3080 | patricklodder | Reviewer | 5 | 
| #1986 | itseasy21 | Author | 5 | Code cleanup
| #1986 | kalvdans | Reviewer | 5 | 
| #1986 | chromatic | Reviewer | 5 | 
| #1986 | patricklodder | Reviewer | 5 | 
| #3139 | patricklodder | Author | 5 | zlib update
| #3139 | xanimo | Reviewer | 5 | Gitian build
| #3139 | chromatic | Reviewer | 5 | Gitian build
| #3141 | xanimo | Author | 5 | Qt patch
| #3141 | patricklodder | Reviewer | 5 | Gitian build
| #3148 | omahs | Author | 5 | fr translation
| #3148 | thisiskeanvy | Reviewer | 5 | 
| #3102 | chromatic | Author | 5 | RPC enhancement
| #3102 | xanimo | Reviewer | 5 | 
| #3102 | patricklodder | Reviewer | 5 | 
| #3157 | practicalswift | Author | 5 | Code cleanup
| #3157 | patricklodder | Author | 5 | Backport author
| #3157 | chromatic | Reviewer | 5 | 
| #3157 | alamshafil | Reviewer | 5 | 
| #3136 | patricklodder | Author | 5 | Safety framework for experimental features
| #3136 | chromatic | Reviewer | 5 | 
| #3136 | edtubbs | Reviewer | 5 | 
| #3146 | patricklodder | Author | 5 | Bugfix on top of #3071
| #3146 | xanimo | Reviewer | 5 | 
| #3146 | edtubbs | Reviewer | 5 | 
| #3146 | chromatic | Reviewer | 5 | 
| #3169 | alamshafil | Author | 5 | Bugfix on depends system
| #3169 | chromatic | Reviewer | 5 | 
| #3169 | patricklodder | Reviewer | 5 | 
| #3191 | chromatic | Author | 5 | Release process fixation in code
| #3191 | patricklodder | Reviewer | 5 | 
| #2676 | chromatic | Author | 5 | Pack NotoSans as additional fallback font
| #2676 | patricklodder | Reviewer | 5 | 
| #2676 | slightlyskepticalpotat | Reviewer | 5 | 
| #3144 | xanimo | Author | 5 | Update macOS helper libs in depends system
| #3144 | patricklodder | Reviewer | 5 | Gitian build
| #3144 | chromatic | Reviewer | 5 | Gitian build
| #3207 | bjacquin | Author | 5 | Bugfixes on Qt and Boost integration
| #3207 | chromatic | Reviewer | 5 | 
| #3207 | patricklodder | Reviewer | 5 | 
| #3213 | patricklodder | Author | 5 | Bugfixes on includes
| #3213 | xanimo | Reviewer | 5 | 
| #3213 | chromatic | Reviewer | 5 | 
| #3217 | bwarden | Author | 5 | Log verbosity
| #3217 | chromatic | Reviewer | 5 | 
| #3224 | anish-m-code | Author | 5 | Bugfix on Qt transaction confirmation popup
| #3224 | patricklodder | Reviewer | 5 | 
| #3286 | chromatic | Author | 5 | Bugfix for payment tests
| #3286 | patricklodder | Reviewer | 5 | 
| #3214 | patricklodder | Author | 5 | Bugfix on top of #3071
| #3214 | chromatic | Reviewer | 5 | 
| #3214 | xanimo | Reviewer | 5 | 
| #3235 | chromatic | Author | 5 | RPC enhancement
| #3235 | patricklodder | Reviewer | 5 | 
| #3299 | alamshafil | Author | 5 | RPC enhancement
| #3299 | edtubbs | Reviewer | 5 | 
| #3299 | patricklodder | Reviewer | 5 | 
| #3301 | patricklodder | Author | 5 | Code cleanup
| #3301 | xanimo | Reviewer | 5 | 
| #3301 | chromatic | Reviewer | 5 | 
| #3306 | alamshafil | Author | 5 | Tests for #3299
| #3306 | patricklodder | Reviewer | 5 | 
| #3322 | xanimo | Author | 5 | Implement getmocktime
| #3322 | chromatic | Reviewer | 5 | 
| #3322 | patricklodder | Reviewer | 5 | 
| #3308 | patricklodder | Author | 5 | Fix macOS brew build
| #3308 | chromatic | Reviewer | 5 | 
| #3308 | michilumin | Reviewer | 5 | 
| #3239 | anish-m-code | Author | 5 | Paper wallet printing fix (windows)
| #3239 | chromatic | Reviewer | 5 | 
| #3239 | patricklodder | Reviewer | 5 | 
| #3358 | patricklodder | Author | 5 | Reduce Qt footprint
| #3358 | xanimo | Reviewer | 5 | Gitian build
| #3358 | chromatic | Reviewer | 5 | Gitian build
| #3357 | edtubbs | Author | 5 | protobuf update
| #3357 | chromatic | Reviewer | 5 | 
| #3335 | ajaxpop | Author | 5 | Bugfix on includes
| #3335 | chromatic | Reviewer | 5 | 
| #3335 | patricklodder | Reviewer | 5 | 
| #3345 | edtubbs | Author | 5 | zlib update
| #3345 | xanimo | Reviewer | 5 | Gitian build
| #3345 | patricklodder | Reviewer | 5 | Gitian build
| #3345 | chromatic | Reviewer | 5 | Gitian build
| #3364 | edtubbs | Author | 5 | fontconfig update
| #3364 | chromatic | Reviewer | 5 | Gitian build
| #3364 | patricklodder | Reviewer | 5 | Gitian build
| #3271 | sinetek | Author | 5 | expat update
| #3271 | chromatic | Author | 1 | Maintainer rebase
| #3271 | xanimo | Reviewer | 5 | Gitian build
| #3271 | patricklodder | Reviewer | 5 | Gitian build
| #3257 | chromatic | Author | 5 | Enhance walletnotify
| #3257 | sinetek | Reviewer | 5 | 
| #3257 | patricklodder | Reviewer | 5 | 
| #3389 | patricklodder | Author | 5 | Fee estimation fix
| #3389 | chromatic | Reviewer | 5 | 
| #3389 | victorsk2019 | Reviewer | 5 | 
| #3382 | patricklodder | Author | 5 | Test for #3257
| #3382 | chromatic | Reviewer | 5 | 
| #3395 | fultondoge | Author | 5 | ko_KR translation
| #3395 | serious-gemini | Reviewer | 5 | 
| #3395 | hwanyoungchoi | Reviewer | 5 | 
| #3297 | alamshafil | Author | 5 | Backport of getblockstats
| #3297 | chromatic | Reviewer | 5 | 
| #3297 | edtubbs | Reviewer | 5 | 
| #3297 | patricklodder | Reviewer | 5 | 
| #3297 | slightlyskepticalpotat | Reviewer | 5 | 
| #3412 | chromatic | Co-Author | 5 | Runtime BIP70 disable
| #3412 | patricklodder | Co-Author | 5 | 
| #3412 | michilumin | Reviewer | 5 | 
| #3388 | anish-m-code | Author | 5 | rpcuser backport
| #3388 | patricklodder | Reviewer | 5 | 
| #3388 | chromatic | Reviewer | 5 | 
| #3413 | patricklodder | Author | 5 | Translation update
| #3413 | chromatic | Reviewer | 5 | 
| #3413 | slightlyskepticalpotat | Reviewer | 5 | 
| #2686 | patricklodder | Author | 5 | macOS build guide
| #2686 | michilumin | Reviewer | 5 | 
| #2686 | fierceskit | Reviewer | 5 | 
| #3422 | patricklodder | Author | 5 | Static seed update
| #3422 | chromatic | Reviewer | 5 | 
| #3422 | slightlyskepticalpotat | Reviewer | 5 | 
| #3419 | slightlyskepticalpotat | Author | 5 | zh_CN translation
| #3419 | marsdoge | Reviewer | 5 | 
| #3430 | fultondoge | Author | 5 | ko_KR translation
| #3430 | ilsubyega | Reviewer | 5 | 
| #3195 | thisiskeanvy | Author | 5 | fr translation
| #3195 | martyornot | Reviewer | 5 | 
| #3433 | patricklodder | Author | 1 | Minor enhancement to #3389
| #3433 | chromatic | Reviewer | 5 | 
| #3428 | mishaboar | Author | 5 | it translation
| #3428 | oshawa-connection | Reviewer | 5 | 
| #3435 | cijsk | Author | 5 | nl translation
| #3435 | patricklodder | Reviewer | 5 | 
| #3443 | patricklodder | Author | 5 | RPC maturity documentation
| #3443 | chromatic | Reviewer | 5 | 
| #3434 | fierceskit | Author | 5 | Fedora build guide
| #3434 | chromatic | Reviewer | 5 | 
| #3434 | patricklodder | Reviewer | 5 | 
| #3431 | danielw86dev | Author | 5 | pl translation
| #3431 | mangekyousharingan | Reviewer | 5 | 
| #3411 | chromatic | Co-Author | 5 | WSL build guide
| #3411 | schmeckl3s | Co-Author | 5 | 
| #3411 | martin | Co-Author | 5 | 
| #3411 | mishaboar | Co-Author | 5 | 
| #3411 | fultondoge | Co-Author | 5 | 
| #3411 | edtubbs | Co-Author | 5 | 
| #3411 | fierceskit | Reviewer | 5 | 
| #3441 | daank-c | Author | 5 | fr_CA translation
| #3441 | john-inubook | Reviewer | 5 | 
| #3442 | patricklodder | Author | 5 | Release notes
| #3442 | chromatic | Reviewer | 5 | 
| #3442 | slightlyskepticalpotat | Reviewer | 5 | 
| #3442 | georgeartem | Reviewer | 5 | 
| #3053 | alamshafil | Author | 0 | Gitian key
| #3053 | patricklodder | Reviewer | 0 | 
| #3057 | patricklodder | Author | 1 | Open dev branch
| #3057 | michilumin | Reviewer | 1 | 
| #3063 | alamshafil | Author | 1 | Fix Qt warnings
| #3063 | patricklodder | Reviewer | 1 | 
| #3066 | acerix | Author | 1 | Fix systemd example
| #3066 | patricklodder | Reviewer | 1 | 
| #3070 | daank-c | Author | 1 | README fix
| #3070 | patricklodder | Reviewer | 1 | 
| #3076 | patricklodder | Author | 1 | Includes fix
| #3076 | chromatic | Reviewer | 1 | 
| #3085 | edtubbs | Author | 1 | CI enhancement
| #3085 | michilumin | Reviewer | 1 | 
| #3135 | patricklodder | Author | 1 | Reverts #3091
| #3135 | chromatic | Reviewer | 5 | 
| #3137 | chromatic | Author | 1 | Fix dbwrapper test warning
| #3137 | patricklodder | Reviewer | 1 | 
| #3138 | chromatic | Author | 1 | Optimize is_hex
| #3138 | patricklodder | Reviewer | 1 | 
| #3140 | patricklodder | Author | 1 | CI version bump
| #3140 | chromatic | Reviewer | 1 | 
| #3140 | xanimo | Reviewer | 1 | 
| #3145 | xanimo | Author | 1 | Minor CI fix
| #3145 | patricklodder | Reviewer | 1 | 
| #3143 | dongcarl | Author | 5 | Depends fix for untar
| #3143 | xanimo | Author | 1 | Backport author
| #3143 | patricklodder | Reviewer | 1 | 
| #3154 | patricklodder | Author | 1 | Minor CI fix
| #3154 | chromatic | Reviewer | 1 | 
| #3158 | patricklodder | Author | 0 | Bugfix on #3157
| #3158 | chromatic | Reviewer | 1 | 
| #3186 | jimjimmiejames | Author | 1 | Update rpcuser
| #3186 | chromatic | Reviewer | 1 | 
| #3186 | patricklodder | Reviewer | 1 | 
| #3142 | patricklodder | Author | 1 | Minor enhancement to #3082
| #3142 | chromatic | Reviewer | 5 | 
| #3142 | xanimo | Reviewer | 5 | 
| #3196 | ninvoido | Author | 1 | Copyright update
| #3196 | patricklodder | Reviewer | 1 | 
| #3196 | chromatic | Reviewer | 1 | 
| #3205 | patricklodder | Author | 1 | CI version bump
| #3205 | chromatic | Reviewer | 1 | 
| #3205 | xanimo | Reviewer | 1 | 
| #3228 | anish-m-code | Author | 1 | fix bumpfee.py
| #3228 | patricklodder | Reviewer | 1 | 
| #3233 | jalinwang | Author | 1 | Spelling errors
| #3233 | chromatic | Reviewer | 1 | 
| #3233 | tosufever | Reviewer | 1 | 
| #3241 | anish-m-code | Author | 1 | Fix fee filter test
| #3241 | chromatic | Reviewer | 1 | 
| #3292 | patricklodder | Author | 1 | Update .gitignore
| #3292 | chromatic | Reviewer | 1 | 
| #3307 | alamshafil | Author | 1 | Minor enhancement to #3299
| #3307 | patricklodder | Reviewer | 1 | 
| #3318 | itsajay1029 | Author | 1 | Clean up BTC mentions
| #3318 | patricklodder | Reviewer | 1 | 
| #3318 | slightlyskepticalpotat | Reviewer | 1 | 
| #3309 | patricklodder | Author | 1 | QA logging fix
| #3309 | chromatic | Reviewer | 1 | 
| #3327 | chromatic | Author | 1 | Bitcoin->Dogecoin
| #3327 | patricklodder | Reviewer | 1 | 
| #3332 | ajaxpop | Author | 1 | Code cleanup
| #3332 | chromatic | Reviewer | 1 | 
| #3332 | patricklodder | Reviewer | 1 | 
| #3340 | justinvforvendetta  | Author | 1 | Minor update to windows build guide
| #3340 | patricklodder | Reviewer | 1 | 
| #3355 | patricklodder | Author | 1 | CI job for translation linting
| #3355 | chromatic | Reviewer | 1 | 
| #3383 | patricklodder | Author | 1 | Reduce CIs
| #3383 | chromatic | Reviewer | 1 | 
| #3383 | edtubbs | Reviewer | 1 | 
| #3384 | edtubbs | Author | 1 | Add security note to build docs
| #3384 | patricklodder | Reviewer | 1 | 
| #3386 | daank-c | Author | 1 | satoshi->koinu
| #3386 | patricklodder | Reviewer | 1 | 
| #3386 | michilumin | Reviewer | 1 | 
| #3414 | patricklodder | Author | 1 | Update m4
| #3414 | chromatic | Reviewer | 1 | 
| #3416 | patricklodder | Author | 1 | Update chainparams
| #3416 | chromatic | Reviewer | 1 | 
| #3417 | patricklodder | Author | 1 | Add platform note to install docs
| #3417 | chromatic | Reviewer | 1 | 
| #3420 | chromatic | Author | 1 | Bugfix to #2676
| #3420 | patricklodder | Reviewer | 1 | 
| #3421 | chromatic | Author | 1 | Fix Qt warnings
| #3421 | patricklodder | Reviewer | 1 | 
| #3423 | patricklodder | Author | 1 | Manpage update
| #3423 | chromatic | Reviewer | 1 | 
| #3423 | slightlyskepticalpotat | Reviewer | 1 | 
| #3427 | patricklodder | Author | 1 | Qt4 bugfix
| #3427 | chromatic | Reviewer | 5 | 
| #3427 | slightlyskepticalpotat | Reviewer | 5 | 
| #3429 | chromatic | Author | 1 | Strict cli parsing for rpc tests
| #3429 | patricklodder | Reviewer | 1 | 
| #3429 | slightlyskepticalpotat | Reviewer | 1 | 
| #3444 | patricklodder | Author | 1 | Update debian/copyright doc
| #3444 | chromatic | Reviewer | 1 | 
| #3447 | patricklodder | Author | 1 | Update inline copyright notices
| #3447 | chromatic | Reviewer | 1 | 
| #3448 | patricklodder | Author | 1 | Release fixation
| #3448 | chromatic | Reviewer | 1 | 
| #3448 | slightlyskepticalpotat | Reviewer | 1 | 

I will provide a CSV the coming weekend, in case there are any corrections.

**Comment by georgeartem** [comment]
Awesome 🙏

**Comment by daank-c** [comment]
Looks good to me!

**Comment by georgeartem** [comment]
Ok 👍 Sent from my iPhoneOn Apr 2, 2024, at 6:46 PM, daanksy ***@***.***> wrote:﻿
Looks good to me!

**Comment by 0foxes** [comment]
Looks good to me! Just one thing: does this mean that we currently have
multiple active tipjars and funds that contribute towards Dogecoin
development?

I was under the impression that there was one old dev-fund controlled by
several core devs (patricklodder, rnicoll, and langerhans if memory serves)
that was transferred to the Dogecoin Foundation after some drama a few
years ago, and that there are no other similar sources of funding.

Thanks for indulging my curiosity...

**Reply by patricklodder** [reply]
> does this mean that we currently have multiple active tipjars and funds that contribute towards Dogecoin development?

The tips from the old tipjar have been split into 2 distinct containers, which was the middle-ground resolution for the "drama" you mention. I expect both of these to pay out for this release.

I'm not aware of any other active funds at this moment besides some bounties, which can use this as a reference.

**Reply by 0foxes** [reply]
I see, I assume you mean these two

https://dogechain.info/address/AC8Q9Z4i4sXcbW7TV1jqrjG1JEWMdLyzcy

https://dogechain.info/address/9xEP9voiNLw7Y7DS87M8QRqKM43r6r5KM5

**Comment by chromatic** [comment]
I've reviewed all the commits listed and :+1: to the weights.

**Comment by Schmeckl3s** [comment]
Oh hey I'm a Dogecoin Dev/Contributor/Co-Author now. 
Let's Go!

**Comment by omahs** [comment]
Hey @rnicoll, I just saw my contribution here and your reddit post. Could you add my address?

My address: DQLkCHGKgVSMefpq2ZhcmMa4CHXufybKyw

**Comment by danielw86dev** [comment]
Same, thx
